src/main/webapp/index.html

```diff
@@ -8,30 +8,32 @@
   <script src="https://apis.google.com/js/platform.js" async defer></script>
   <!-- Content URL obtained from credentials section under gcp dashboard -->
   <meta name="google-signin-client_id" content="757099697912-i6jll98mfgochdo2vgjcovf64pepjesc.apps.googleusercontent.com">
-
   <script type="module" src="scripts/index-script.js" defer></script>
+  <link rel="stylesheet" href="styles/general.css">
   <script type="module" src="scripts/nav-bar.js"></script>
-
   <link rel="stylesheet" href="styles/homestyle.css">
 </head>
 <body>
-  <div id="nav-placeholder"> </div>
+  <div id="nav-bar">
```

Comment:
Will this div "nav-bar" be around the nav bar on all pages?

Reply:
yes it needs to be there for styling purposes

Reply:
so everypage will need to have

  <div id="nav-bar">
    <div id="nav-placeholder"> </div>
  </div>

Reply:
<div id="nav-bar">
    <div id="nav-placeholder"> </div>
  </div>